### PR TITLE
Adjust release string

### PR DESCRIPTION
**Pull request by LukasReschke**
cc @blizzz As discussed. Set the version to 9.0.50 to prevent migration problems from ownCloud.
cc @karlitschek Mind a review? THX.

### CODE REVIEW

**Comment by blizzz** [comment]
LGTM, as discussed, would love to hear others opinions, too.

**Comment by mar1u5** [comment]
We keep 9.0.0 as "human readable"?

File: version.php [hunk 1]
```diff
@@ -26,10 +26,10 @@
 // We only can count up. The 4. digit is only for the internal patchlevel to trigger DB upgrades
 // between betas, final and RCs. This is _not_ the public version number. Reset minor/patchlevel
 // when updating major/minor version number.
-$OC_Version = array(9, 0, 3, 0);
+$OC_Version = array(9, 0, 50, 0);
 
 // The human readable string
-$OC_VersionString = '9.0.0 alpha';
+$OC_VersionString = '9.0.0';
```
Comment (MorrisJobke) [inline version.php]: As mentioned by @mar1u5 I would also change this to 9.0.50

Comment (LukasReschke) [inline version.php]: Why? This is what is displayed to the user.

Comment (blizzz) [inline version.php]: i second Lukas

**Comment by MorrisJobke** [comment]
:+1:

**Pull request merged**